doc/source/user_guide/groupby.rst

```diff
@@ -184,8 +187,8 @@ only verifies that you've passed a valid mapping.
 .. note::
 
    Many kinds of complicated data manipulations can be expressed in terms of
-   GroupBy operations (though can't be guaranteed to be the most
-   efficient). You can get quite creative with the label mapping functions.
+   GroupBy operations (it can't be guaranteed to be the most efficient implementation).
```

Comment:
The parenthetical remark sounds awkward to me with "though" removed.

Reply:
I sounds awkward before and after my changes, I think. 
I'll change it to "(though it can't be be guaranteed to be the most efficient implementation)". Unless you tell me it's not OK.

Reply:
That sounds good to me.